packages/auth/src/api/index.test.ts

```diff
@@ -306,6 +315,22 @@ describe('api/_performApiRequest', () => {
       );
       expect(mock.calls[0].request).to.eql(request);
     });
+
```

Comment:
nit: you could add a `context("with non-Firebase Errors")` and do the mockFetch override. That way you can keep the beforeEach in the above tests.

Reply:
Done